projects/components/src/menu-dropdown/menu-item/menu-item.component.scss

```diff
@@ -2,9 +2,5 @@
 
 .menu-item {
   @include popover-menu-item;
-
-  &.disabled {
-    opacity: 0.4;
-    cursor: not-allowed;
-  }
+  @include disableable;
```

Comment:
I love this word so much.

Reply:
I'm ashamed. I almost did something like `supports-disable`, but I figured it's better to push the limits of science.